socs/agents/ucsc_radiometer/agent.py

```diff
@@ -73,6 +73,9 @@ def acq(self, session, params=None):
             except ValueError:
                 pm.sleep()
                 continue
+            except requests.exceptions.ConnectionError:
```

Comment:
Great, we should also include a `self.log.warn()` level message about the closed connection so that it's clear from the logs that the radiometer web server is unavailable.